### DISCUSSION TITLE
RE VR games perform terribly sometimes...... HDR?

### DISCUSSION THREAD

**Discussion by davidflana** [praydog/REFramework#932] [General Questions] [answered]
Hello!
Great mod first off all!  I have played through RE2, RE3, and now RE7.  I am using a Lenovo Legion 7i with Quest 2.  Sometimes when I start the game (any of them), it runs slow, and there are these weird shapes on the screen, and my laptop fan is not screaming like it usually does...  It seems like the game is not getting priority or something. (nothing else is running.  I bought this PC to run these VR games and nothing else.).  Sometimes, if I hit ALT-TAB to break out of the full screen, it fixes it and everything speeds up.  Other times this does not fix it.  I read that enabling HDR can cause problems, so I made sure that was off.  I also found that I can trigger the problem when I change ammo type (another snag in the game).  The only way I can change ammo is to window the game (alt-tab) and then press the G key on the keyboard.  SOMETIMES when I do this, it messes the game up and I get the slow down.  Then I have to completely reboot the Quest 2 and the PC to get it working again.  PLEASE someone give me the golden answer to this annoying thorn in the side of an otherwise AWESOME experience!!

**Comment by praydog** [comment]
Turn off Hardware Accelerated GPU Scheduling

**Reply by davidflana** [reply]
thank you for the fast response!  Tried that and rebooted.  Still the same issue....  any other ideas?

**Reply by davidflana** [reply]
right now it seems to be stuck in this slow performance issue (it was before I changed the setting you mentioned above so that didn't cause me to be stuck).  Even after rebooting everything, it is staying slow even when I ALT-TAB out of full screen....  The dead giveaway is that the fan inside the laptop slows down.  Normally when I load the game, the fan revs up really high and makes a lot of noise.  It is quiet when this issue is happening.  What is strange is, If I play Halflife Alyx, everything works great, every time.....The fan is screaming loud, and the game is fast and fluid.  It is only the modded RE games that do this sporadically.  Like I said, I finished RE2 and RE3, and I am several hours into RE7 now.  I know it works, there must be some setting somewhere that is kicking in....  Thanks again for your time!!!

**Reply by praydog** [reply, marked as answer]
Look up how to rename/disable SteamVR game theater. Apparently a recent SteamVR update has been forcing this on, destroying the framerate.

This doesn't apply if you are using OpenXR with a non-SteamVR runtime.

**Reply by davidflana** [reply]
you are a genius!  She's running smooth as glass!  Once again you solved my problem quickly.  Thank you kind sir!

**Comment by Sawtooth76** [comment]
I got the same problem. Can you tell me how to change to a non steam vr runtime... and which one?
Thanks a bunch!

**Reply by davidflana** [reply]
The answer to my problem was to disable theater mode in SteamVR by renaming the exe file.  Those instructions are here:

https://steamcommunity.com/app/250820/discussions/0/1692669912408365970/

**Reply by Sawtooth76** [reply]
Good to know there is a second solution.
I got it working with another runtime. Someone wrote a tool to switch it easily.
https://github.com/WaGi-Coding/OpenXR-Runtime-Switcher/releases/
